ironfish/src/rpc/routes/wallet/index.ts

```diff
@@ -5,14 +5,15 @@
 export * from './addTransaction'
 export * from './burnAsset'
 export * from './create'
+export * from './createAccount'
 export * from './createTransaction'
```

Comment:
sorting of endpoints

Reply:
a bit of an aside, but I wonder if there's a way that we can get lint to enforce this. otherwise I fear we're likely to add a new endpoint out of order at some point in the future 😅